### PR TITLE
Add com.ramussoftware.ramus

### PR DESCRIPTION
**Pull request by imnkkr**
Ramus is an application to model business processes using IDEF0 methodology.

### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
     - "--filesystem=home": the app doesn't use portals, so it needs full access to home directory
     - "--env=PATH=...": adding java to path
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
    - The project doesn't seem to be actively developed and the only distribution method is an archive in *Releases* without an installer, so submitting all these files seems pointless right now

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: why x86_84 since it's in Java?

File: com.ramussoftware.ramus.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        "install -Dm755 com.ramussoftware.ramus -t /app/bin/",
+        "install -Dm644 com.ramussoftware.ramus.svg -t /app/share/icons/hicolor/scalable/apps/",
+        "install -Dm644 com.ramussoftware.ramus.metainfo.xml -t /app/share/metainfo/",
+        "install -Dm644 com.ramussoftware.ramus.desktop -t /app/share/applications/",
+        "install -Dm644 com.ramussoftware.ramus-mime.xml /app/share/mime/packages/com.ramussoftware.ramus.xml"
+      ]
+    },
+    {
+      "name": "ramus",
+      "buildsystem": "simple",
```
Comment (hfiguiere) [inline com.ramussoftware.ramus.json:58]: this should all be one module.

File: com.ramussoftware.ramus.desktop
Comment (hfiguiere) [inline com.ramussoftware.ramus.desktop:1]: this should be submitted upstreanm. it's not flatpak specific

File: com.ramussoftware.ramus-mime.xml
Comment (hfiguiere) [inline com.ramussoftware.ramus-mime.xml:1]: this should be submitted upstreanm. it's not flatpak specific

File: com.ramussoftware.ramus.metainfo.xml
Comment (hfiguiere) [inline com.ramussoftware.ramus.metainfo.xml:1]: this should be submitted upstreanm. it's not flatpak specific

**Comment by hfiguiere** [comment]
> The project doesn't seem to be actively developed and the only distribution method is an archive in Releases without an installer, so submitting all these files seems pointless right now

it's NEVER pointless.

File: flathub.json (hunk 1 above)
Comment (imnkkr) [reply flathub.json:2]: Makes sense, actually

**Comment by imnkkr** [comment]
Thank you! I'll try upstreaming it first before submitting the package

**Pull request closed without merging**